### DISCUSSION TITLE
EXP1 and EXP2 ports

### DISCUSSION THREAD

**Discussion by v-flashpoint** [sanni/cartreader#408] [Need help]
Hi all,

I placed an order on AliExpress a few weeks ago for everything on the "What to Order" page, and most of it has now come through, but now I'm looking through the build instructions again, I've noticed the EXP1 and EXP2 ports on the Main PCB. I don't see anything quite listed on "What to Order" page that matches the description of those ports, so I just wanted to check - do I need to buy those from somewhere, or are they going to come on the PCB/as part of one of the other bits from AliExpress? I've had most of the stuff come through now, but I can't see anything that looks quite like those ports...

Thanks,
Ian

**Comment by sanni** [comment]
I see, thanks for pointing it out. The older build was using male pin headers(cut down from the 1x40 pin headers) and the new is using [2x5 IDC box headers](https://www.aliexpress.com/item/1005001400147026.html) so that the display cable can't be plugged in the wrong way by accident. I forgot to update the shopping list, fixed now.

If you bought the male pin headers you can use them like in the picture on the left side, just be mindful when plugging in the display cables, the little notch on the cable end points towards the top where the boxed pin headers have the cutout:

![IMG_1411](https://user-images.githubusercontent.com/19169494/179350170-89dbd9ec-8b1a-418d-8431-401ea783b6de.jpg)

**Reply by v-flashpoint** [reply]
Thank you! Good to know. I did indeed buy the 1x40, so will have a go at making something like in the pic (and just make sure I connect it the right way round :) )
